docs/design/2021-12-09-TiDB-log-based-incremental-backup-draft-1.md

```diff
@@ -0,0 +1,495 @@
+# TiDB Log-based Incremental Backup
+
+- Author(s): @kennytm
```

Comment:
👀

Reply:
yep, most of the content is just copy & paste from *TiDB Log-based Incremental Backup (Draft 1)*...